girder/api/rest.py

```diff
@@ -805,21 +805,16 @@ def route(self, method, route, handler, nodoc=False, resource=None):
                 'WARNING: No access level specified for route %s %s' % (
                     method, routePath))
 
-    def removeRoute(self, method, route, handler=None, resource=None):
+    def removeRoute(self, method, route, resource=None):
         """
         Remove a route from the handler and documentation.
 
-        .. deprecated :: 2.3.0
-
         :param method: The HTTP method, e.g. 'GET', 'POST', 'PUT'
         :type method: str
         :param route: The route, as a list of path params relative to the
                       resource root. Elements of this list starting with ':'
                       are assumed to be wildcards.
         :type route: tuple[str]
-        :param handler: The method called for the route; this is necessary to
-                        remove the documentation.
-        :type handler: Function
```

Comment:
In the code below, what if `route` is not found within `nLengthRoutes`? Then `handler` will never be defined, yet the code afterwards still references it.

@danlamanna may hate this, but this a perfect case for a `for ... else` expression.